platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -20,55 +72,55 @@
 
 @property (nonatomic) BOOL iconOptional;
```

Comment:
Should we add a custom getter, so `-isIconOptional`? Or would it be too difficult to ensure that all the getters remain grammatically correct?